common/nym-lp/Cargo.toml

```diff
@@ -14,13 +14,23 @@ bytes = { workspace = true }
 dashmap = { workspace = true }
 sha2 = { workspace = true }
 ansi_term = { workspace = true }
+log = { workspace = true }
 utoipa = { workspace = true, features = ["macros", "non_strict_integers"] }
 rand = { workspace = true }
+# rand 0.9 for KKT integration (nym-kkt uses rand 0.9)
+rand09 = { package = "rand", version = "0.9.2" }
```

Comment:
while using old versions is not an ideal choice for new crates, but is there anything in particular blocking us from using rand 0.8 in `nym-kkt` given we control its content?